components/inbound-endpoints/org.wso2.carbon.inbound.endpoint.persistence/src/main/java/org/wso2/carbon/inbound/endpoint/persistence/PersistenceUtils.java

```diff
@@ -77,10 +85,13 @@
     private static final QName HTTPSPROTOCOL_QN = new QName(HTTPSPROTOCOLS_ATT);
     private static final QName REVOCATIONVERIFIER_QN = new QName(REVOCATIONVERIFIER_ATT);
     private static final QName PREFERRED_CIPHERS_QN = new QName(PREFERRED_CIPHERS_ATT);
-
+    private static final String PORT_OFFSET_SYSTEM_VAR = "portOffset";
+    private static final String PORT_OFFSET_CONFIG = "Ports.Offset";
```

Comment:
Any reason for the capitalization P and O. Usually key are lowercase in properties file. Also better to have a singular noun.